rtublen/Studentportalen
# Add profile page for Andre Paulsson

Adds profile page for André Paulsson

## How to verify change

1. Start application.
2. Open /profiles/andrepaulsson
3. Verify that you see profile page for André Paulsson
